packages/next/client/dev/error-overlay/format-webpack-messages.js

```diff
@@ -111,14 +111,17 @@ function formatMessage(message, verbose) {
   }
 
   // Add helpful message for users trying to use Sass for the first time
-  if (lines[1] && lines[1].match(/Cannot find module.+node-sass/)) {
+  if (lines[1] && lines[1].match(/Cannot find module.+sass/)) {
     // ./file.module.scss (<<loader info>>) => ./file.module.scss
     lines[0] = lines[0].replace(/(.+) \(.+?(?=\?\?).+?\)/, '$1')
```

Comment:
```suggestion
    const firstLine = lines[0].split('!')
    lines[0] = firstLine[firstLine.length-1]
```

This seems to correctly print both global and module CSS file names without anything else.